Aquality.Selenium.Core/src/Aquality.Selenium.Core/Applications/Startup.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Aquality.Selenium.Core.Configurations;
+using Aquality.Selenium.Core.Elements;
+using Aquality.Selenium.Core.Elements.Interfaces;
+using Aquality.Selenium.Core.Localization;
+using Aquality.Selenium.Core.Logging;
+using Aquality.Selenium.Core.Utilities;
+using Aquality.Selenium.Core.Waitings;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Reflection;
+
+namespace Aquality.Selenium.Core.Applications
+{
+    /// <summary>
+    /// Allows to resolve dependencies for all services in the Aquality.Selenium.Core library
+    /// </summary>
+    public class Startup
```

Comment:
Maybe rename to 'ServicesResolved'?

Reply:
ASP.NET Core apps use a Startup class, which is named Startup by convention. 
[Thus, I'm not completely following this convention]
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/startup